project_purchase_link/models/project_project.py

```diff
@@ -23,84 +21,205 @@ class ProjectProject(models.Model):
     )
 
     def _compute_purchase_info(self):
+        # read group
+        query = self.env["purchase.order.line"]._search(
```

Comment:
Why removing the `read_group`? I don't think the complexity of the code deserves the few juice on the performance (if any).

Reply:
OK, I see... Everything gets complicated with the new analytic system. I let users of this module to judge, as I don't use it and only doing some administrative collaboration.